buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -459,7 +459,7 @@ class BeamModulePlugin implements Plugin<Project> {
         google_auth_library_credentials             : "com.google.auth:google-auth-library-credentials:$google_auth_version",
         google_auth_library_oauth2_http             : "com.google.auth:google-auth-library-oauth2-http:$google_auth_version",
         google_cloud_bigquery                       : "com.google.cloud:google-cloud-bigquery:1.108.0",
-        google_cloud_bigquery_storage               : "com.google.cloud:google-cloud-bigquerystorage:0.120.1-beta",
+        google_cloud_bigquery_storage               : "com.google.cloud:google-cloud-bigquerystorage:0.125.0-beta",
```

Comment:
Linkage Checker detected that its grpc-api 1.25 is not compatible with grpc-core 1.27. https://issues.apache.org/jira/browse/BEAM-5898?focusedCommentId=17053644&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17053644